README.md

```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       ```
 
 
+## **December 23 - Exam Seating Arrangement**
+  - **Problem**
+    - Exams happen all the time in our education system and a huge amount of effort is put into planning the seating arrangement to prevent students form malpractices. The task here is to create a system that does this automatically when the number of departments, the number of classrooms, and the students roll numbers are given and automatically generate the output plan.
+    - There is only one condition when it comes to seating the students is that **No two students of the same department should sit together**
+
+
+  - **Example**
+    ```
+    Number of classrooms: 2
+    Size of classrooms (1-50): 16
+    Enter the number of departments (2-10): 2
+    Department 1 Code: CS
+    Department 2 code: EC
+    Students in department 1 (1-100): 15
+    Starting roll no: 1
+    Students in department 2 (1-100): 16
+    Starting roll no: 1
+    ```
+    ```
+    ROOM 1
+    CS1   EC1   CS2   EC2
+    EC4   CS4   EC3   CS3
+    CS5   EC5   CS6   EC6
+    EC8   CS8   EC7   CS7
+
+    ROOM 2
+    CS9   EC9   CS10    EC10
+    EC12  CS12  EC11    CS11
+    CS13  EC13  CS14    EC14
+    EC8   ____  EC15    CS15
+    ```
+
+
+## **December 24 - Reverse a String Using Recursion**
+  - **Problem**
+    - In simple words, Recursion means a function calling itself.
```

Comment:
Recursion simply refers to a function which calls itself. Recursive functions have two parts - a base case or stopping scenario and something that calls the same function with modified parameters. Check out the resources section for some great explanations of how it works. 

Once you understand recursion, you will find yourself able to break down and reduce the complexity of a program much more easily. While they certainly don't replace loops, they are excellent in certain scenarios.

For this question, you will have to recursively reverse a string. First write a program to reverse a string normally. Then try to convert it into a recursive function.